packages/flutter/lib/src/gestures/events.dart

```diff
@@ -232,6 +233,8 @@ abstract class PointerEvent with Diagnosticable {
   }) : localPosition = localPosition ?? position,
        localDelta = localDelta ?? delta;
 
+  /// Corresponds to the motion event that resulted in this pointer event. (only android)
+  final int motionEventId;
```

Comment:
Is this a unique ID for each `PointerEvent` or can multiple PointerEvents have the same motionEventId?

Reply:
Nope, this is unique per `PointerEvent`. Multiple `PointerEvent`s can not have the same `motionEventId`.